packages/mds-compliance/api.ts

```diff
@@ -122,7 +122,7 @@ function api(app: express.Express): express.Express {
             .includes(policy.policy_id)
         ) {
           const [geographies, deviceRecords] = await Promise.all([
-            db.readGeographies(),
+            db.readGeographies() as Promise<Geography[]>,
```

Comment:
We should probably figure out a nice way to specify the return type based on the input params... Is that something we can do with TS @drtyh2o? We could also just write a typeguard to verify that these are indeed `Geography`'s rather than `GeographySummary`'s